frontend/src/utils/api/reviews.js

```diff
@@ -29,8 +29,8 @@ export function reviewMapper({ id, code, title, body, likes, numResponses, userL
     recommend: Boolean(recommend),
     numResponses,
     user,
-    published: formatDistanceStrict(timestamp * 1000, new Date(), { addSuffix: true }),
-    timestamp: timestamp
+    published: formatDistanceStrict(new Date(timestamp), new Date(), { addSuffix: true }),
```

Comment:
No it needs to be a backend fix pretty sure this won’t work on safari. Time needs to be in Unix epoch time.